pyxodr/road_objects/road.py

```diff
@@ -552,6 +576,35 @@ def junction_connecting_ids(self) -> Dict[str, Set[int]]:
 
         return _junction_connecting_ids
 
+    @cached_property
+    def road_properties(self) -> RoadProperties:
+        """Return all the existing road properties for this road."""
+        length = float(self["length"])
+        name = self["name"]
+        type_element = self["type"]
+
+        max_speed = None
+        if type_element is not None:
+            speed_element = type_element["speed"]
+            if speed_element is not None:
+                try:
```

Comment:
I think we can remove the generic try block because if the input is OpenDRIVE-compliant I don't see anything that would trigger an excpetion.